src/util.cpp

```diff
@@ -72,8 +97,10 @@ QString Util::formatString(const QString& input, bool replaceEmptyString)
 
 QString Util::formatSymbol(const Data::Symbol& symbol, bool replaceEmptyString)
 {
-    return formatString(Settings::instance()->prettifySymbols() ? symbol.prettySymbol : symbol.symbol,
-                        replaceEmptyString);
+    const QString format =
+        formatString(Settings::instance()->prettifySymbols() ? symbol.prettySymbol : symbol.symbol, replaceEmptyString);
+    return Settings::instance()->collapseTemplates() ? collapseTemplate(format, Settings::instance()->collapseDepth())
```

Comment:
this is wrong, we don't want to collapse the whole formatted string, only the symbol, i.e.:

```
symbolString = prettifySymbols ? symbol.prettySymbol : symbol.symbol;
if (collapseTemplates)
    symbolString = collapsteTemplate(symbolString, depth);
return format(symbolString, replaceEmptyString);
```